### ISSUE TITLE
Allow user to opt-in to follow up emails

### ISSUE THREAD

**Issue by yoavaviram** [your-digital-rights/yourdigitalrights.org#160]
On the Personal Information Form allow the user to opt-in to the following follow up emails: 
- emails guiding the user through the rest of the request process
- monthly newsletter with further recommendations regarding opt-outs

The UI must warn the user that this will involve us processing the user's email address, and explain that we will obtain it by adding an email address to the BCC field.

Designs: https://zpl.io/2GdQNlj
Figma file: https://www.figma.com/file/YUaVKI5U2wR8Qhrqyexoyr/yourdigitalright?node-id=0%3A999

**Comment by roughprada** [comment]
@yoavaviram I'm adding the form with the difference that, instead of adding an extra checkbox, we are making the selection of reminder part of the submission form. I am aware we need to have 1 of the radios selected by default, shall we default to the declining one?

And secondly, if you can have a look at the copy, it will be awesome.

**Comment by roughprada** [comment]
Designs here.
![form](https://user-images.githubusercontent.com/5957010/111884886-505c1100-89bc-11eb-81e2-d9e12077ecb7.png)

I am moving the designs to Figma, are you both familiar with it? we can share links if easier that way.

**Comment by yoavaviram** [comment]
Fine with Figma. 

What happens if the user checks the newsletter but not the follow up? The warning about data collection is still relevant.

**Comment by yoavaviram** [comment]
@roughprada can you please share a link where I can leave comments on the design (Figma)?

**Comment by yoavaviram** [comment]
Just a reminder that newsletter is out of cope for now.

**Comment by roughprada** [comment]
@yoavaviram I have to do a little bit of housekeeping on the files, they are not in the best state at the moment. You have received an invitation regardless, and you should be able to double click on whichever piece of text and edit it. Please let me know if  you cannot access it or need a bit of guidance until we have it better organised: https://www.figma.com/file/YUaVKI5U2wR8Qhrqyexoyr/yourdigitalright?node-id=0%3A999

**Comment by roughprada** [comment]
@yoavaviram here the zeplin link: zpl://screen?pid=5b07cf7b9af8b1243f584d16&sid=5e26f59a46e5cf7deaa8ada3
@honzie let me know  if you have an email I can add you to zeplin

**Comment by yoavaviram** [comment]
Not sure what is the thing you pasted for a zeplin link, but it's not a URL

**Comment by honzie** [comment]
I've added this copy and the BCC functionality in https://github.com/your-digital-rights/yourdigitalrights.org/pull/173. I'd appreciate feedback—especially since this is my first PR. @roughprada , you may be interested in reviewing and clarifying the copy.

**Comment by roughprada** [comment]
@yoavaviram yes! sorry, I've added the handle to the app, sorry!

**Comment by roughprada** [comment]
Awesome @honzie ! I think we will need to add spacing between fields as it's very heavy on copy now.
I'd leave the copy review to @yoavaviram but I wonder if we can make changes to the separation between fields? Something like 40px desktop and 20px mobile? I'd try to also tweak the font. The form is missing a bit of hierarchy (this is not new, historical issues!)

**Comment by honzie** [comment]
@roughprada : definitely! Where were you thinking of adding the space? On top of the "As stated in legislation..." and/or on top or below each text input? This should be very straightforward to do. And I can roll this with any hierarchy-enhancing tweaks if you have any that are ready to implement.

**Comment by roughprada** [comment]
@honzie I believe there might be a little refactoring to do at some point, but the idea is that between fields we have a 40px separation (25px on mobile), but where (top/bottom) I  am not too sure, wherever you consider it makes more sense. I sent you an invitation to the Figma file so you can check the spacing. Please bear in mind that we recently moved the project from sketch to Figma, so things will be a little all over the place.

I need to work a little more on the hierarchy-enhancing tweaks, I'll make a ticket as soon as is ready, thanks!
